.python-version

```diff
@@ -1 +1 @@
-3.11
+3.13
```

Comment:
Only python 3.12 runtime is available

https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

```suggestion
3.12
```

Reply:
cc @paulschwarzenberger what do you think of updating the python version from 3.11 to 3.12?

Reply:
Now python 3.13 is supported

https://aws.amazon.com/blogs/compute/python-3-13-runtime-now-available-in-aws-lambda/